File: src/main/java/com/bombproj/model/ProjectFile.java
package com.bombproj.model;

import com.queryflow.annotation.Table;
import lombok.Data;

import java.util.Date;

@Data
@Table("project_file")
public class ProjectFile {

    private String id;
    private String fileName;
    private String suffix;
    private Long size;
    private String parentId;
    private String creator;
    private Date createTime;
    private Date updateTime;
    private Integer type;
    private Integer state;
    private Integer starred;
    private Integer shared;
    private String sharedPassword;
    private String sharedId;
    private Long version;
    private String projectId;
    private String recycledBy;
    private String deletedBy;

}

File: src/main/scala/com/bombproj/model/ProjectFile.scala
package com.bombproj.model

import java.util.Date

import com.queryflow.annotation.Table

import scala.beans.BeanProperty

@Table("project_file")
case class ProjectFile() {

    @BeanProperty var id: String = _
    @BeanProperty var fileName: String = _
    @BeanProperty var suffix: String = _
    @BeanProperty var size: Long = _
    @BeanProperty var parentId: String = _
    @BeanProperty var creator: String = _
    @BeanProperty var createTime: Date = _
    @BeanProperty var updateTime: Date = _
    @BeanProperty var `type`: Int = _
    @BeanProperty var state: Int = _
    @BeanProperty var starred: Int = _
    @BeanProperty var shared: Int = _
    @BeanProperty var sharedPassword: String = _
    @BeanProperty var sharedId: String = _
    @BeanProperty var version: Long = _
    @BeanProperty var projectId: String = _
    @BeanProperty var recycledBy: String = _
    @BeanProperty var deletedBy: String = _

}
